modules/drivers/velodyne/fusion/pri_sec_fusion_component.cc

```diff
@@ -64,6 +64,9 @@ bool PriSecFusionComponent::Proc(
     }
     std::this_thread::sleep_for(std::chrono::milliseconds(5));
   }
+  auto diff = Time::Now().ToNanosecond() -
+              static_cast<uint64_t>(target->header().lidar_timestamp());
```

Comment:
Seems like that it is `uint64_t`. So the `static_cast` is not needed.

Reply:
Yes, you are right. google::protobuf::uint64 is an alias for uint64_t. Have fixed.